### ISSUE TITLE
Failed to load the atom-memolist-new package

### ISSUE THREAD

**Issue by ghost** [tgfjt/atom-memolist-new#4]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.0.2
**System**: Mac OS X 10.10.4
**Thrown From**: [atom-memolist-new](https://github.com/tgfjt/atom-memolist-new) package, v0.3.0
### Stack Trace

Failed to load the atom-memolist-new package

```
At Cannot find module 'atom-space-pen-views'

Error: Cannot find module 'atom-space-pen-views'
  at Module._resolveFilename (module.js:328:15)
  at Function.Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app.asar/src/module-cache.js:383:52)
  at Function.Module._load (module.js:270:25)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/Users/maximillian/.atom/packages/atom-memolist-new/index.coffee:1:29)
  at Object.<anonymous> (/Users/maximillian/.atom/packages/atom-memolist-new/index.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Package.module.exports.Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:709:34)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:188:28
  at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:165:15)
  at Package.module.exports.Package.load (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:179:12)
  at PackageManager.module.exports.PackageManager.loadPackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:372:14)
  at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:451:30)
  at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-manager.js:368:29
  at exit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-manager.js:59:16)
  at triggerExitCallback (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:213:47)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:227:18
  at Socket.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:98:18)
  at emitOne (events.js:82:20)
  at Socket.emit (events.js:166:7)
  at Pipe.close (net.js:464:12)
  at EventEmitter.ipc.sendSync (/Applications/Atom.app/Contents/Resources/atom.asar/renderer/api/lib/ipc.js:21:31)
  at BrowserWindow.RemoteMemberFunction [as getPosition] (/Applications/Atom.app/Contents/Resources/atom.asar/renderer/api/lib/remote.js:110:29)
  at Atom.module.exports.Atom.getWindowDimensions (/Applications/Atom.app/Contents/Resources/app.asar/src/atom.js:530:29)
  at Atom.module.exports.Atom.storeDefaultWindowDimensions (/Applications/Atom.app/Contents/Resources/app.asar/src/atom.js:563:25)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:104:21
  at handler (/Applications/Atom.app/Contents/Resources/app.asar/src/space-pen-extensions.js:115:32)
  at jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/space-pen/vendor/jquery.js:4681:9)
  at elemData.handle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/space-pen/vendor/jquery.js:4359:46)

```
### Commands

```
```
### Config

``` json
{}
```
### Installed Packages

``` coffee
# User
atom-memolist-new, v0.3.0
color-picker, v2.0.11
linter-squirrel, v0.3.0
minimap, v4.12.1
travis-ci-status, v0.16.0
vim-mode, v0.57.0

# Dev
No dev packages
```

**Comment by tgfjt** [comment]
Thanks for uncovering it.:ocean: 

It seems to be my bad... I'll fix soon.

**Comment by tgfjt** [comment]
@silentotto 
Fixed, and atom-memolist-new@0.4.0 was published.
